test/statetest/statetest_loader.cpp

```diff
@@ -395,11 +395,6 @@ void validate_deployed_code(const state::State& state, evmc_revision rev)
                         " is invalid: " + std::string(get_error_message(result)));
                 }
             }
-            else
```

Comment:
Why do we need this change?

Reply:
This was committed as part of https://github.com/ethereum/evmone/pull/696.